webcompat/api/endpoints.py

```diff
@@ -264,14 +265,8 @@ def modify_labels(number):
 def get_repo_labels():
     '''XHR endpoint to get all possible labels in a repo.
     '''
-    request_headers = get_request_headers(g.request_headers)
-    if g.user:
-        path = 'repos/{0}/labels'.format(REPO_PATH)
-        labels = github.raw_request('GET', path, headers=request_headers)
-    else:
-        path = 'https://api.github.com/repos/{0}/labels'.format(REPO_PATH)
-        labels = proxy_request('get', uri=path, headers=request_headers)
-    return (labels.content, labels.status_code, get_headers(labels))
+    with open( STATIC_PATH + '/config_data/labels.json') as f:
+        return (f.read(), 200, {'content-type': 'application/json'})
```

Comment:
We have a constant `JSON_MIME` in this file that you can use instead of the `application/json` string.